### PR TITLE
Enhance Czech Translations in cs.json for Mammotion

### PR DESCRIPTION
**Pull request by virusczz**
### **User description**
update to 0.1.21

___

### **Description**
- Enhanced Czech translations in `cs.json` for better user experience.
- Added translations for Bluetooth connection status and cutting mode states.
- Improved naming for device tracking.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>cs.json</strong><dd><code>Update Czech translations for Mammotion component</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

custom_components/mammotion/translations/cs.json
<li>Added new translation for Bluetooth connection status.<br> <li> Introduced new cutting mode states in the translation.<br> <li> Enhanced device tracker name translation.<br>

</details>

  </td>
  <td><a href="https://github.com/mikey0000/Mammotion-HA/pull/109/files#diff-a4dc9af1bd0eea823a85fb82158b50ae1d8c364a90029d6f7ff87fef51f2eb8d">+15/-5</a>&nbsp; &nbsp; </td>
</tr>
</table></td></tr></tr></tbody></table>

### CODE REVIEW

**Comment by mikey0000** [comment]
@virusczz same here, thanks!

**Comment by virusczz** [comment]
done

**Review by mikey0000 — APPROVED** [review]

**Comment by virusczz** [comment]
done hope that will be ok :) we will see :)

File: custom_components/mammotion/translations/cs.json [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
       "no_longer_present": "Zařízení již není k dispozici",
       "not_supported": "Zařízení není podporováno",
-      "no_device_name": "Pro pokračování je vyžadován název zařízení",
       "reconfigure_successful": "Znovu nastavení bylo úspěšné"
     },
     "flow_title": "Nastavení sekačky Mammotion",
     "step": {
       "bluetooth_confirm": {
         "description": "Nastavení {name}"
+        "stay_connected_bluetooth": "Udržovat Bluetooth připojený"
```
Comment (mikey0000) [inline custom_components/mammotion/translations/cs.json:17]:
```suggestion
        "data": {
          "stay_connected_bluetooth": "Udržovat Bluetooth připojený"
        }
```

**Pull request merged**